site/content/entry/yuzu-migration/index.md

```diff
@@ -0,0 +1,81 @@
++++
+date = "2019-10-04T23:51:00+05:30"
+title = "yuzu - The Migration"
+author = "CaptV0rt3x"
+forum = 150132
++++
+
+A very good day to you all ***yuz-ers*** out there! 
+We are currently in the process of merging both the Canary and Nightly versions of yuzu into a single release channel.
+Jump right in to find out more about this!
+<!--more-->
+***
+
+## Announcement
+
+<article class="message">
+<div class="message-header is-dropdown">
+<p>We are merging yuzu nightly and yuzu canary releases into a single "yuzu" release!</p>
+</div>
+</article>
+
+Don't fret, we are just doing some reorganization to make things easier for both the users and the developers.
+Today marks the retirement of both yuzu nightly and yuzu canary release channels.
```

Comment:
Don't we typically capitalize `Nightly` and `Canary` when referencing them officially?